### ISSUE TITLE
DT3 fails to open DMC-GX1 RW2 files

### ISSUE THREAD

**Issue by ygu** [darktable-org/darktable#11781]
**Describe the bug/issue**
Impossible to open pictures taken with a Panasonic Lumix DMC-GX1. In the light table, they are represented with a space invader, and when I try to open one of them, I get the following message (translated from french): "darktable didn't succeed in loading ...RW2 ... check your camera is supported". I can see in the terminal:
```
[rawspeed] (P1150594.RW2) virtual std::unique_ptr<rawspeed::RawDecoder> rawspeed::RawParser::getDecoder(const rawspeed::CameraMetaData*), line 91: No decoder found. Sorry.
```
I've checked, and yes, that camera is supported by DT.

**To Reproduce**
1. Open DT (it opens on the light table, showing the last working directory with RW2 files)
2. Double-click on one of those files
3. See error

**Expected behavior**
The picture should open normally.

**Platform**
_Please fill as much information as possible in the list given below. Please state "unknown" where you do not know the answer and remove any sections that are not applicable _

I tried with darktable 3.0.1 (installed from the official ubuntu repositories) and with darktable 3.8.1 (installed from the ppa:ubuntuhandbook1/darktable)
* OS                : Linux - kernel 5.4.0-109-generic
* Linux - Distro    : Ubuntu 20.04.4 LTS
* Memory            : 15,5 Go
* Graphics card     : Mesa Intel HD Graphics 620 (KBL GT2)

**Additional context**
It works well with JPEG files.

**Comment by Nilvus** [comment]
Could you try by installing darktable 3.8.1 from open build repository (check on install page in darktable.org). ubuntuhandbook is not officially supported and it's better to be sure it's not from a wrong build package (already seen) even if it's seems improbable that this could be the cause. It would also be good if you can share an affected RAW file here to test.

**Comment by ygu** [comment]
I've installed DT 3.8.1 and I have the same problem. The only difference is that with this version, the error is indicated in the UI.
Here is an example of affected RAW: https://e1.pcloud.link/publink/show?code=XZf3vzZSHIbMLNvXP7y7vRzFzrSNk4mDCuk

Here are the error messages, in the UI and the console:
https://e1.pcloud.link/publink/show?code=XZW3vzZCWvANyI3SOVVYY8SJnVMNYxzO3H7
https://e1.pcloud.link/publink/show?code=XZg3vzZN4dkkn7lE2pHVRjRiInnWjxVgdV0

I tried on another machine with linux 5.15.0-27-lowlatency with a ubuntu-studio 22.04.
With the same RAW, same problem.

**Comment by Nilvus** [comment]
@ygu: I've tested your RAW and first thing is that I find it quite small for a RAW (1Mb). I check the file correctly and it's a JPG (with RW2 extension on filename but a JPG). So it's normal that you have that issue. I suggest you to check carefully files properties. I've used gthumb app to check properties (I'm on Gnome). Another simple way to test is to rename file to change extension from RW2 to jpg. Then try to open it with darktable and you will see that it's ok. I tested also a real RW2 for a DMC-GX1 from raw.pixls.us and the RAW opens correctly.

So you probably have some issues with your files, not with darktable fails to open DMC-GX1 raw files. I let this issue open few days waiting for your feedback.
